docs/DeveloperGuide.md

```diff
@@ -238,6 +238,33 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Compactable and expandable PersonCard
+
+This feature is accomplished by recording the current state of the `PersonCard` with a boolean variable called `isExpanded`. `isExpanded` is true if `PersonCard` is expanded and false if `PersonCard` is compacted. `isExpanded` is set to `false` by default so that the application starts with all `PersonCard` compacted.
+
+There is also a listener that listens to mouse clicks on each `PersonCard`. Clicking on a `PersonCard` calls one of two methods depending on its `isExpanded` value. If `isExpanded` is `true`, it calls `hideLabels()`. If `isExpanded` is `false`, it called `showLabels()`.
```

Comment:
Typo? If `isExpanded` is `false`, it **calls** `showLabels()`.

Reply:
Looking at the code, I think it is correct. If the current state is not expanded then it will show the labels, thus changing state from not expanded to expanded.

Reply:
Hmm I meant that the word `called` in `If isExpanded is false, it called showLabels()` should be `calls`. My bad! My wording was ambiguous.